.github/workflows/ci.yml

```diff
@@ -20,10 +20,12 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - name: Install custom tap
+    - name: Set up Homebrew
+      uses: Homebrew/actions/setup-homebrew@master
+    - name: Add our custom tap
       run: |
-        mkdir -p /usr/local/Homebrew/Library/Taps/$GITHUB_REPOSITORY
-        cp -r * /usr/local/Homebrew/Library/Taps/$GITHUB_REPOSITORY/
+        mkdir -p "$(brew --repository)/Library/Taps/$GITHUB_REPOSITORY"
```

Comment:
Nit: It shouldn't be necessary to create the directly to then create a symlink

Reply:
Good spot! Changed in https://github.com/torchbox/homebrew-tap/pull/7/commits/b77d162f019905a85906868a166daacf7a046fc0